### PR TITLE
Update Planet Installation (Fixes #2016)

### LINKED ISSUE #2016: Docker log
### Problem
During the installation of planet in step 2. The user are not fully aware when to press Ctrl+ c to stop the logs. 

### Proposed solution
The user needs to press Ctrl+z only when the certain code is seen in the screen as in the screenshot below:

issue 1

After seeing this screenshot, the user can press Ctrl+c then move on to the next process.

### PR DESCRIPTION
**Pull request by janamr**
Fixes #2016 

### Description
Additional instruction based on Issue #2016 which was given the 'okay' but not yet changed.
Plus I fixed the gray code box from previous PR #2036 by pressing 'Enter' after the first ```

### RawGit preview link
https://rawgit.com/janamr/janamr.github.io/patch-1/#!pages/vi/vi-planet-installation-and-configuration.md

### CODE REVIEW

File: pages/vi/vi-planet-installation-and-configuration.md [hunk 1]
```diff
@@ -125,6 +125,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
   - `docker tag treehouses/planet:latest treehouses/planet:local`
   - `docker tag treehouses/planet:db-init treehouses/planet:db-init-local` 
 
+6. To exit the logs press `CTRL+C` when you see the following code: 
+![db-init-img](https://user-images.githubusercontent.com/41874655/44394481-03d79d00-a557-11e8-91ed-378eccc95011.jpg)
```
Comment (svlesiv) [inline pages/vi/vi-planet-installation-and-configuration.md]: Hi, @janamr looks great! However, probably better to store image inside 'images' folder of the project and have a link like this: `db-init-img`.
a5f466497ac4e67aca5dfd130b697c937db5b06c1a957a9bc4 pimgpsh_fullsize_distr

Also, you could create an empty line after '....the following code:' before image, in that way number 6 in the list will not appear dislocated.

Comment (janamr) [reply pages/vi/vi-planet-installation-and-configuration.md]: Hey thanks @svlesiv I was wondering about that. I'll fix it now.

Comment (dogi) [reply pages/vi/vi-planet-installation-and-configuration.md]: @svlesiv I agree with both points - thanks

**Comment by svlesiv** [comment]
Hi @janamr, you don't have to create a separate pull request for the image. Probably better to start from the beginning.
http://open-learning-exchange.github.io/#!./pages/vi/vi-github-issues.md

1. Update your master branch on your local repository.
     * git fetch upstream
     * git checkout master
     * git merge upstream/master
     * git push origin master
2. Create a branch for this issue and switch to that branch.
    * git checkout -b name_of_your_new_branch
3. Download the image and put it inside the images folder.
4. Adjust the code.
     * Your image link might look like this:  `db-init-img`
     * other changes ......
5. Create a commit.
     * git status
     * git add .
     * git commit -m "your_commit_message (fixes #issue_no)"
     * git push -u origin name_of_your_new_branch 

Please let me know if you have any questions. Also, you can create some other issues to work on.

**Pull request closed without merging**